src/main/java/org/opentripplanner/graph_builder/module/osm/FinlandWayPropertySetSource.java

```diff
@@ -0,0 +1,63 @@
+package org.opentripplanner.graph_builder.module.osm;
+
+import org.opentripplanner.routing.edgetype.StreetTraversalPermission;
+
+/**
+ * OSM way properties for Finnish roads. FinlandWayPropertySetSource is derived from NorwayPropertySetSource by seime
+
+ * The main difference compared to the default property set is that most of the highway=trunk roads also allows walking and biking, 
+ * where as some does not. 
+ * http://wiki.openstreetmap.org/wiki/Tag:highway%3Dtrunk
+ * http://wiki.openstreetmap.org/wiki/Highway:International_equivalence
+ *
+ *
+ * @author juusokor
+ * @see WayPropertySetSource
+ * @see DefaultWayPropertySetSource
+ */
+public class FinlandWayPropertySetSource implements WayPropertySetSource {
+
+  @Override
+  public void populateProperties(WayPropertySet props) {
+    // Replace existing matching properties as the logic is that the first statement registered takes precedence over later statements
+    props.setProperties("highway=trunk_link", StreetTraversalPermission.ALL, 2.06,
+        2.06);
+    props.setProperties("highway=trunk", StreetTraversalPermission.ALL, 7.47, 7.47);
+
+    // Don't recommend walking in trunk road tunnels
+    props.setProperties("highway=trunk;tunnel=yes", StreetTraversalPermission.CAR, 7.47, 7.47);
+
+    // Do not walk on "moottoriliikennetie"
+    props.setProperties("motorroad=yes", StreetTraversalPermission.CAR, 7.47, 7.47);
+
+    // Remove informal and private roads
+    props.setProperties("highway=*;informal=yes", StreetTraversalPermission.NONE);
+    props.setProperties("highway=service;access=private", StreetTraversalPermission.NONE);
+    props.setProperties("highway=trail", StreetTraversalPermission.NONE);
```

Comment:
Should these be in the default?

Reply:
We came to the conclusion, that they shouldn't.